pkg/preparer/setup.go

```diff
@@ -255,10 +266,11 @@ func (c *PreparerConfig) getOpts() (consul.Options, error) {
 	}
 
 	return consul.Options{
-		Address: c.ConsulAddress,
-		HTTPS:   c.ConsulHttps,
-		Token:   token,
-		Client:  client,
+		Address:  c.ConsulAddress,
+		HTTPS:    c.ConsulHttps,
+		Token:    token,
+		Client:   client,
+		WaitTime: c.ConsulConfig.WatchWaitTime,
```

Comment:
https://github.com/hashicorp/consul/blob/master/consul/rpc.go#L378. Although point taken, i'll make this explicitly 5 minutes if the setting is < 5 minutes